### PR TITLE
Day adding, leap year methods

### PR DESCRIPTION
**Pull request by wozane**

### CODE REVIEW

File: day_adding/test_leap_year.rb [hunk 1]
```diff
@@ -8,18 +8,18 @@
...
 class TestLeapYear < Minitest::Test
   def test_if_2016_is_leap_year
-    assert year_leap?(2016)
+    assert Calendar.year_leap?(2016)
   end
 
-  def test_that_1900_is_not_eap_year
-    assert !year_leap?(1900)
+  def test_that_1900_is_not_leap_year
+    assert_equal(false, Calendar.year_leap?(1900))
```
Comment (esse) [inline day_adding/test_leap_year.rb]: it would be better to use `assert !Calendar.year_leap?(1900)` instead

File: day_adding/calendar.rb [hunk 2]
```diff
@@ -26,15 +14,34 @@ def add_one_day(year, month, day)
...
+      false
+    end
+  end
+
+  def self.add_one_day(year, month, day)
+    day_count = DAYS_IN_MONTH[month]
+
+    case
+    when month == 2 && year_leap?(year)
+      if day == 28
```
Comment (esse) [inline day_adding/calendar.rb]: `if` inside `case` doesn't look very well. I think it apparently would be better to use `if` form here. 

However, you've learned pretty well how `case` works 👍

Comment (wozane) [reply day_adding/calendar.rb]: well noted

**Comment by wozane** [comment]
@esse request for review before merge

**Review by esse — APPROVED** [review]

**Pull request merged**